contrib/win32/openssh/libssh.vcxproj

```diff
@@ -417,7 +417,6 @@
       <ExcludedFromBuild Condition="$(UseOpenSSL)==false">true</ExcludedFromBuild>
     </ClCompile>
     <ClCompile Include="$(OpenSSH-Src-Path)ssh-ed25519.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)ssh-pkcs11.c" />
```

Comment:
Please keep this in libssh project. This aligns with the upstream implementation 
https://github.com/openssh/openssh-portable/blob/df93529dd727fdf2fb290700cd4f1adb0c3c084b/Makefile.in#L106

Reply:
Added it back. Thanks!